lucille-core/src/main/java/com/kmwllc/lucille/stage/EmitNestedChildren.java

```diff
@@ -0,0 +1,43 @@
+package com.kmwllc.lucille.stage;
+
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import java.util.Iterator;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This stage emits attached children documents, removing them from the parent document. Will skip document if no children is found.
+ *
+ * Config Parameters:
+ * - drop_parent (Boolean): if set to true, will mark parent document as dropped
+ *
+ */
+public class EmitNestedChildren extends Stage {
+
+  private final boolean dropParent;
+  private static final Logger log = LoggerFactory.getLogger(EmitNestedChildren.class);
+
+  public EmitNestedChildren(Config config) {
+    super(config, new StageSpec().withRequiredProperties("drop_parent"));
+    this.dropParent = config.getBoolean("drop_parent");
```

Comment:
CreateChildrenStage calls this property "dropParent"
CollapseChildrenDocuments has a "dropChildren" property
Looking at stage properties, we currently have some stages using CamelCase and some using underscores. That's a bigger issue than we should tackle in this PR, just noting it here as a reminder to create a separate ticket for it. I'll create the ticket.

Reply:
we already had this ticket for stage property name standardization: https://kmwllc.atlassian.net/browse/LC-501